mainstyle.css

```diff
@@ -20,52 +39,89 @@ img {
 .aboutus {
     text-align: center;
     width: 100%;
-    background-color: pink;
+    background-color: #ff8cd8;
     display: block;
+    width:auto;
+    height: auto;
+    margin-top: 40px;
+    margin-left: 15%;
+    margin-right: 15%;
+    border: solid;
+    border-width: medium;
+    font-family: 'Pangolin', cursive;
+    box-shadow: -10px -10px 5px black;
+    background-image: linear-gradient(#3a4d6d, #ff8cd8)
   }
 
-/*.leftmenu {
-  background-color: green;
-  float: left;
-  width: 500px;
+/*.middle{
+  display: inline;
+}*/
+
+.locations {
+  background-color: #444d5b;
+  position: absolute;
+  width: 350px;
   height: 680px;
   font-size: 40px;
   text-align: center;
   padding: 15px;
   margin: 50px;
-  display:block;
+  display:inline-block;
+  margin-left: 18%;
+  margin-bottom: 200px;
+  font-family: 'Pangolin', cursive;
+  border: solid;
+  box-shadow: -5px -5px 5px black;
+  background-image: linear-gradient(#f73894, #8cb0ea)
 }
 
-.leftmenubackground {
-  background-color: red;
-  float: left;
-  width: 550px;
-  height: 720px;
-  font-size: 40px;
-  text-align: center;
-  padding: 15px;
-  margin: 50px;
-  display: block;
-}
 
-#rightmenu {
+
+.storehours {
+  background-color: #444d5b;
   float: right;
-  width: 500px;
-  height: 680px;
+  width: 350px;
+  height:680px;
   font-size: 40px;
   text-align: center;
   padding: 15px;
   margin: 50px;
-  background-color: blue;
+  display:block;
+  margin-right: 18%;
+  margin-bottom: 200px;
+  font-family: 'Pangolin', cursive;
```

Comment:
If you just put this on the body, all the child elements will inherit that font-family.